.github/workflows/node.js.yml

```diff
@@ -9,8 +9,8 @@ on:
       - "frontend/**"
   pull_request:
     paths:
-      - "frontend/**"
-
+       - "frontend/**"
+       - ".github/workflows/node.js.yml"
```

Comment:
I dont see why it’s necessary to rerun the workflow on this very specific file. You probably only left this for testing. Recommend to remove

Reply:
this is for if we modify the yml file itself @farisdurrani

Reply:
but if we need to remove it, we can

Reply:
I just don't want to make a precedent all github workflow files are triggered if the specific workflow file is changed. It's an odd convention

Reply:
ok. so we should remove that line @farisdurrani